pkg/controller/discovery/web/plan.go

```diff
@@ -290,11 +266,107 @@ func (p *PlanPods) buildPods(h *PlanHandler, ref *v1.ObjectReference) ([]Pod, er
 			return nil, err
 		}
 		for _, model := range podModels {
-			pod := Pod{}
+			pod := PlanPod{}
 			pod.With(model, &cluster)
 			pods = append(pods, pod)
 		}
 	}
 
 	return pods, nil
 }
+
+//
+// Plan REST resource.
+type Plan struct {
+	// The k8s namespace.
+	Namespace string `json:"namespace"`
+	// The k8s name.
+	Name string `json:"name"`
+	// Raw k8s object.
+	Object *migapi.MigPlan `json:"object,omitempty"`
+}
+
+//
+// PlanPods REST resource.
+type PlanPods struct {
+	// The k8s namespace.
+	Namespace string `json:"namespace"`
+	// The k8s name.
+	Name string `json:"name"`
+	// List of controller pods.
+	Controller []PlanPod `json:"controller"`
+	// List of relevant pods on source cluster.
+	Source []PlanPod `json:"source"`
+	// List of relevant pods on the destination cluster.
+	Destination []PlanPod `json:"destination"`
+}
+
+//
+// Container REST resource.
+type Container struct {
+	// Pod k8s name.
+	Name string `json:"name"`
+	// The URI used to obtain logs.
+	Log string `json:"log"`
+}
+
+//
+// PlanPod REST resource.
```

Comment:
The `PlanPod` and `Container` is a _plan_ thing to renamed and moved from `pod.go`.  This allows for the standardized `Pod` resource in `pod.go`.